ansibullbot/triagers/plugins/docs_info.py

```diff
@@ -0,0 +1,306 @@
+import ast
+import dataclasses
+from itertools import filterfalse
+import logging
+import re
+
+import requests
+
+RE_DOCS_PATTERNS = [
```

Comment:
@acozine Should we include `lib/ansible/plugins/doc_fragments/` as well?

Reply:
Also, if bot determines that a PR is `docs_only` and only other changes in addition to docs only are in `changelogs/fragments`, we might add the label anyway?